spec/models/user.spec.js

```diff
@@ -168,4 +168,93 @@ describe("User model", () => {
       });
     });
   });
+
+  it("returns two users if either match username or email", (done) => {
```

Comment:
Do you mean this is just trying to sign up two separate users and check if the credentials are different?

Reply:
Or is this bit about signing in with either a username or email?